R/rsc_filter.R

```diff
@@ -20,6 +19,9 @@ rsc_filter <- function(req, res) {
                            fixed = TRUE
       )
     }
+
+    # Ensure path starts with '/'
+    if (!stringi::stri_startswith(content_path, "/")) content_path <- paste0("/", content_path)
     req$content_path <- content_path
   }
 
```

Comment:
I'd highly recommend not using the path from the header... What happened to querying the vanity URL?

Reply:
IMO, querying the vanity URL introduces additional overhead for every call, since we have to call the Connect API instead of just relying on information already present in the header. Is there a reason we shouldn't trust the header?

Reply:
We decided to use the header in favor of querying the API